toptal/picasso
# feat(FormLabel): remove asterisk, add (optional)

[FX-1229](https://toptal-core.atlassian.net/browse/FX-1229)

### Description

Remove the asterisk from FormLabel and add `(optional)` for unrequired fields.

### How to test

- Build and open storybook
- Verify required Form Labels don't have an asterisk
- Verify unrequired Form Labels have an (optional) suffix
- Verify Form Label with explicit `required={false}` don't have a suffix

### Screenshots

![image](https://user-images.githubusercontent.com/254110/98243122-2dcddc00-1f76-11eb-895a-4f98955cda52.png)


### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run  whole pipeline
- `@toptal-bot run danger` - Danger checks
- `@toptal-bot run lint` - Run linter
- `@toptal-bot run test` - Run jest
- `@toptal-bot run build` - Check build
- `@toptal-bot run test:visual` or `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
